### PR TITLE
Added screenshots to Model Builder install

### LINKED ISSUE #13090: Add screen shots for how to install/ uninstall from VS
Add screenshots of the steps listed.

[AB#1596631](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1596631)
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5692cfb5-52e1-b6b9-2b87-7ea0decc2882
* Version Independent ID: 2da762a6-5bc4-a8d1-afae-f1a8243eee33
* Content: [How to install Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder#feedback)
* Content Source: [docs/machine-learning/how-to-guides/install-model-builder.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/how-to-guides/install-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by luisquintanilla**
Added screenshots to Model Builder install

[Preview Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder?branch=pr-en-us-14510)

Fixes #13090

### CODE REVIEW

**Comment by natke** [comment]
Are the screenshots in this PR?

(Re: the numbering, it's better to use 1. - these will be rendered as an HTML numbered list and numbered accordingly)

**Comment by luisquintanilla** [comment]
They are, maybe it's just having trouble rendering. Sometimes I can see them. I updated the lists and maybe that also helps rendering.

File: docs/machine-learning/how-to-guides/install-model-builder.md [hunk 1]
```diff
@@ -40,25 +40,25 @@ ML.NET Model builder can be installed either through the Visual Studio Marketpla
 
 1. In the menu bar, select **Tools** > **Extensions and Updates**
 
-    ![VS201701](./media/install-model-builder/VS201701.png)
+    ![VS2017 open manage extensions dialog](./media/install-model-builder/VS201701.png)
```
Comment (natke) [inline docs/machine-learning/how-to-guides/install-model-builder.md]: Oh sorry I wasn't clear. I meant the image filenames

Comment (natke) [reply docs/machine-learning/how-to-guides/install-model-builder.md]: Also best practice is to not use upper case filenames with git

Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/install-model-builder.md]: #resolved

**Review by natke — COMMENTED** [review]
LGTM. Do you want to run this by @rustd, to make sure it's what he had in mind?

**Review by natke — APPROVED** [review]

**Pull request merged**